brush/src/components/Map/style.css

```diff
@@ -0,0 +1,3 @@
+.canvasWrapper {
+  overflow-x: auto;
```

Comment:
Suggestion: Maybe we would add a shadown inside of the card if we have an overflow, to give to user the idea that he can scroll on left.